esp/src/eclwatch/QuerySetDetailsWidget.js

```diff
@@ -214,10 +214,10 @@ define([
                 this.workunitsTab.set("title", newValue);
             } else if (name === "Suspended") {
                 dom.byId(this.id + "SuspendImg").src = newValue ? dojoConfig.getImageURL("suspended.png") : "";
+            } else if (name === "Suspended") {
```

Comment:
Good catch but the API responds with "Suspended:true" if it is suspended by user therefore, I throw in a different icon for that. But, now that I look at this diff I see that Logical Files fix should not be in there I already committed that.